blog/_posts/2017-02-22-repl-0.6-highlights.md

```diff
@@ -0,0 +1,102 @@
+Even so, the REPL in Julia 0.6 includes quite a few new visual tweaks and features.
+In this blog post, we look into some of these changes and customizations to the Julia REPL, available in 0.6.
+
+## Prompt pasting
+
+It is quite common (as in the Julia manual and in subsequent code blocks of this blog post) to write snippets of Julia code like they were copied straight from a REPL session, for example:
+
+```julia
+julia> f() = rand(10^6)
+f (generic function with 1 method)
+
+julia> @time f();
+  0.052948 seconds (25.05 k allocations: 8.769 MiB)
+```
+
+If you wanted to try out this code in 0.5, you would typically copy the code into a text editor, remove the `julia> ` prompt, remove the output, and then finally have code that looked like:
+
+```julia
+f() = rand(10^6);
+
+@time f();
+```
+
+which could then be pasted back into the REPL to be executed.
+In 0.6, the REPL comes with a feature called *prompt pasting* by which a pasted expression starting with `julia> ` automatically has prompts and eventual output scrubbed before being executed.
+If you have a 0.6 REPL currently at hand, you can try to copy and paste the example with the prompt and output above directly into the REPL and you will see that it works in exactly the same way as if we had manually removed them.
+
+Unfortunately, for technical reasons (no support for bracket pasting), this feature does not work in the standard cmd-terminal on Windows.
+There are however multiple alternative terminal emulators on Windows that do support this, e.g. [Console2](https://sourceforge.net/projects/console/) or [Cygwin](https://www.cygwin.com/).
+
+## Printing of stacktraces
+
+When an error occurs, the REPL prints the exception that was thrown along with a [stack trace](https://en.wikipedia.org/wiki/Stack_trace): a list tracing back through the sequence of function calls and source-code lines that were being executed when the error occurred. In previous versions of Julia, however, the stack trace might omit some function calls that were [inlined](https://en.wikipedia.org/wiki/Inline_expansion), making it harder to understand. In Julia 0.6, however, the stack trace includes function calls that were inlined. Unfortunately, in combination with the Julia idiom of breaking complex tasks into many smaller functions, this can sometimes lead to long stack traces. Furthermore, the stack trace includes the type signature of the methods being called, but these type signatures can sometime be quite verbose (e.g. for union types or types taking many parameters). Julia 0.6 greatly improves the readability of this output by coloration, numbering, and interactive features described below.
+
+In earlier julia versions, stacktraces were being printed without much structure.
```

Comment:
Perhaps simplify "stacktraces were being printed" to "stacktraces were printed"?